chapters/5-processing.tex

```diff
@@ -100,32 +107,37 @@ \subsection{Unique identifiers}
   reflecting only that information'',
   depending on the context.
 	This applies to data acquired from partners as well as
-	primary data collected by the research team.}
-It is also possible for a raw dataset to not include a unique identifier,
-or for the identifier to involve a long string
-that is difficult to work with, such as a person's name.
-In such cases, cleaning begins by either carefully creating a variable
-that uniquely identifies the data,
-or merging the ID variable from the master dataset
+	original data collected by the research team.
+}
+
+It is also possible for a raw dataset to not include an unique identifier,
+or that the identifier is not a suitable \textbf{project ID}.\sidenote{
+	\textbf{Project ID:} The main ID used in your project to identify
+	observations.
+	You should never have multiple project IDs for the same unit of observation.
+	The project ID must uniquely and fully identify all observations in the project.
+	See \url{https://dimewiki.worldbank.org/ID\_Variable\_Properties} for more details.}
+Suitable project IDs should, for example, not involve long strings
+that are difficult to work with, such as a person's name,
+or be an ID that is known outside the research team.
+In such cases, cleaning begins by 
+adding a project ID to the raw data.
+If a project ID already exists, 
+for this unit of observation,
+then you should carefully merge it 
```

Comment:
the sidenote about terminology (merge vs join) currently on line 166 should move up here, as we use the 'term' merge from here on

Reply:
done in 93480410479541b119828baa56f6dedc28ca3967